Gemfile

```diff
@@ -6,13 +6,15 @@ source "https://rubygems.org"
 
 gem "bootsnap", require: false, platform: :mri
 
-gem "actionmailer", "< 7.1"
-gem "actionpack", "< 7.1"
-gem "actionview", "< 7.1"
-gem "activemodel", "< 7.1"
-gem "activerecord", "< 7.1"
-gem "activesupport", "< 7.1"
-gem "railties", "< 7.1"
+%w[
+  actionmailer
+  actionpack
+  actionview
+  activemodel
+  activerecord
+  activesupport
+  railties
+].each { |name| gem name, "~> 7.1.0" }
```

Comment:
I think this will break dependabot's parsing of the Gemfile 👀

Reply:
You’re probably right, the dependabot code seems to rely on its own logic and not on the bundler API.